tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -2583,29 +2586,30 @@ void SwiftLangSupport::findRelatedIdentifiersInFile(
         std::vector<ResolvedLoc> ResolvedLocs = resolveRenameLocations(
             Locs.getLocations(), /*NewName=*/StringRef(), *SrcFile, Diags);
 
-        for (auto ResolvedLoc : ResolvedLocs) {
+        assert(ResolvedLocs.size() == Locs.getLocations().size());
+        for (size_t index = 0; index < ResolvedLocs.size(); ++index) {
+          auto RenameLoc = Locs.getLocations()[index];
```

Comment:
`RenameLoc` doesn't appear to be used for anything here, should it be removed?

Reply:
Oh, good catch. Also means that we no longer need `size_t index`.